pkg/w3m

```diff
@@ -0,0 +1,27 @@
+[mirrors]
+http://downloads.sourceforge.net/project/w3m/w3m/w3m-0.5.3/w3m-0.5.3.tar.gz
+
+[vars]
+filesize=2202328
+sha512=43508c76d07b4d8f19c19f975c0b870aeb94abf0744b6128ee01c759d4e409a8b57bc866baeaf990f309ff73e9a7b02ca455d272b1dd0a93fafb8c72b1fe6d14
+pkgver=1
+
+[deps]
+libgc
+gdk-pixbuf
+
+[build]
+patch -p1 < /src/KEEP/w3m-patch.patch
```

Comment:
ouch, this needs to be `$K` , not `/src/KEEP`

Reply:
fixed :p